src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsauditapi/resource/AuditResourceTest.kt

```diff
@@ -143,16 +143,82 @@ class AuditResourceTest : NoQueueListenerIntegrationTest() {
     @ParameterizedTest
     @MethodSource("secureEndpoints")
     internal fun `satisfies the correct role and scope`(uri: String) {
+      doNothing().whenever(auditService).sendAuditEvent(any())
+
       webTestClient.post()
         .uri(uri)
         .headers(setAuthorisation(roles = listOf("ROLE_AUDIT"), scopes = listOf("write")))
         .body(BodyInserters.fromValue(AuditEvent(what = "secureEndpointCheck")))
         .exchange()
         .expectStatus().isAccepted
 
+      verify(auditService).sendAuditEvent(any())
```

Comment:
should normally verify the parameter being passed in instead of `any`

Reply:
Updated